### ISSUE TITLE
Add self service OpenSLO Slack invite site

### ISSUE THREAD

**Issue by ian-bartholomew** [OpenSLO/OpenSLO#43]
### Problem to solve

As we try and build the community, we shouldn't require people to email me to get an invite to the slack channell

### Proposal

Use something like this https://github.com/hangops/inviterbot to make a site that people can go to for self service

**Comment by rajdama** [comment]
How should I set up in my local?

**Comment by ian-bartholomew** [comment]
@rajdama there should be instructions in the linked project

**Comment by rajdama** [comment]
> @rajdama there should be instructions in the linked project

@ian-bartholomew  it is not clearly what should be done.

**Comment by ian-bartholomew** [comment]
@rajdama updated the description to add some details

**Comment by rajdama** [comment]
> @rajdama updated the description to add some details

Ok 👍

**Comment by ian-bartholomew** [comment]
@fearfactor3 no, but i can make a new repo if you need.  just let me know what you need.  Thanks!

**Comment by fearfactor3** [comment]
No, I'm going to build a POC on my side before I present it to you.  I think I have everything I need.  I was trying to see if I needed to follow certain guidelines or not.  This is the first project I'm trying to contribute to.

**Comment by ian-bartholomew** [comment]
@fearfactor3 amazing, thank you!
